hs/src/Reach/JSUtil.hs

```diff
@@ -268,19 +342,69 @@ instance HasJSAnnot JSExpression where
     JSYieldExpression a _ -> a
     JSYieldFromExpression a _ _ -> a
 
+instance RepJSAnnot JSExpression where
+  rjsa a' = \case
+    JSIdentifier _ x -> JSIdentifier a' x
+    JSDecimal _ x -> JSDecimal a' x
+    JSLiteral _ x -> JSLiteral a' x
+    JSHexInteger _ x -> JSHexInteger a' x
+    JSOctal _ x -> JSOctal a' x
+    JSStringLiteral _ x -> JSStringLiteral a' x
+    JSRegEx _ x -> JSRegEx a' x
+    JSArrayLiteral _ x _ -> JSArrayLiteral a' (map (rjsa a') x) a'
+    JSAssignExpression a x y -> JSAssignExpression (rjsa a' a) (rjsa a' x) (rjsa a' y)
+    JSAwaitExpression _ x -> JSAwaitExpression a' (rjsa a' x)
+    JSCallExpression a _ y _ -> JSCallExpression (rjsa a' a) a' (rjsaJSL a' y) a'
+    JSCallExpressionDot a _ y -> JSCallExpressionDot (rjsa a' a) a' (rjsa a' y)
+    JSCallExpressionSquare a _ y _ -> JSCallExpressionSquare (rjsa a' a) a' (rjsa a' y) a'
+    JSClassExpression _ x y _ r _ -> JSClassExpression a' (rjsa a' x) y a' r a'
+    JSCommaExpression a _ y -> JSCommaExpression (rjsa a' a) a' (rjsa a' y)
+    JSExpressionBinary a x y -> JSExpressionBinary (rjsa a' a) (rjsa a' x) (rjsa a' y)
+    JSExpressionParen _ x _ -> JSExpressionParen a' (rjsa a' x) a'
+    JSExpressionPostfix a x -> JSExpressionPostfix (rjsa a' a) (rjsa a' x)
+    JSExpressionTernary a _ y _ r -> JSExpressionTernary (rjsa a' a) a' (rjsa a' y) a' (rjsa a' r)
+    JSArrowExpression x _ y -> JSArrowExpression (rjsa a' x) a' $ jsStmtToConciseBody (rjsa a' $ jsArrowBodyToStmt y)
```

Comment:
I think it is a mistake to do these conversions

Reply:
okay